instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/camel.yaml

```diff
@@ -0,0 +1,262 @@
+---
+rules:
+  - bean: org.apache.camel:context=*,type=context,name=*
+    prefix: camel.context.
+    metricAttribute:
```

Comment:
Why there's no metric attribute "param(name)"? This will work correctly only if there's always at most one "name" matching the pattern. If there are more, the metric values will get clobbered by multiple MBeans attempting to set the metric values.

Reply:
no, in this particular case context and name have the same value, context is without quotes, that's why I preferred it.

Reply:
Got it.